app/scripts/directives/inlineEdit.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('openshiftConsole').component('inlineEdit', {
+  controller: [
+    InlineEdit
+  ],
+  controllerAs: '$ctrl',
+  bindings: {
+    label: '<',
+    pattern: '<',
+    patternErrmsg: '<',
+    value: '<',
+    saveFunc: '<'
+  },
+  templateUrl: 'views/directives/inline-edit.html'
+});
+
+function InlineEdit() {
+  var ctrl = this;
+
```

Comment:
Should we add a `ctrl.saveFunc = ctrl.saveFunc || function(){};`, so that line 38 won't break if no function is passed in?